uiowa/uiowa
# Admissions - Area of Study Stuff

Resolves #2925
Resolves #2927
Resolves #2932
Resolves #2933
Resolves #2816
Resolves #2980
Resolves #2926
Replaces https://github.com/uiowa/uiowa/pull/2759

## To Test

- `blt frontend && blt ds --site=admissions.uiowa.edu`
- As an editor, I can see admissions.local.drupal.uiowa.edu/admin/content/fragments
- As an editor, I can create some stats: admissions.local.drupal.uiowa.edu/fragment/add
- As a webmaster, I can reference up to 3 stats on AoS admissions.local.drupal.uiowa.edu/node/add/area_of_study
- As an anonymous visitor I can see the stat component with the fragment field data.
- Fill out the rest of the fields on the AoS content.
- Title, Subtitle, Featured Image, Stats, Career Outcomes and Student Opportunities, Course Work, Why Iowa, Admissions Requirements (long text) as mocked https://admissions.prod.drupal.uiowa.edu/academics/accounting.
- Scholarships, related links, majors/minors, programs, stories are present but not styled.
